Lib/test/test_set.py

```diff
@@ -2154,6 +2155,30 @@ def test_cuboctahedron(self):
             for cubevert in edge:
                 self.assertIn(cubevert, g)
 
+class TestPySet_Add(unittest.TestCase):
+    def test_set(self):
+        # Test the PySet_Add c-api for set objects
+        s = set()
+        assert _testcapi.pyset_add(s, 1) == {1}
```

Comment:
Please use assertEqual():

```suggestion
        self.assertEqual(_testcapi.pyset_add(s, 1), {1})
```

Same remark for other tests.